radiobutton.java

```diff
@@ -5,16 +5,9 @@
 public class radiobutton {
 
 	public static void main(String[] args) {
-		// TODO Auto-generated method stub
 		System.setProperty("webdriver.gecko.driver ","geckodriver");
-		
 		WebDriver driver=new FirefoxDriver();
-		
-		/*driver.get("https://book2.spicejet.com/");
```

Comment:
keep `driver.get("https://book2.spicejet.com/");` as this might be needed later.